server/rpc.go

```diff
@@ -460,12 +461,16 @@ func orchestratorInfoWithCaps(orch Orchestrator, addr ethcommon.Address, service
 		workerHardware = workerHardwareToNetWorkerHardware(orch.WorkerHardware())
 	}
 
+	aiCapacity := orch.GetAICapacity()
+	capabilities := orch.Capabilities()
+	capabilities.Constraints.PerCapability[uint32(core.Capability_LiveVideoToVideo)].Models["noop"].Capacity = uint32(aiCapacity.ContainersIdle)
```

Comment:
@leszko only thing about `PerCapability` is that we are grouping the capacity by model too, but for live video we don't have any distinction on model right, the containers are just available for any model? Since we always use the comfyui model maybe it's ok to hardcode for now, but we also use noop for testing 🤔

Reply:
Just remembered we list separate models in the config: https://github.com/livepeer/livepeer-infra/blob/master/inventory.yml#L52 so I'll need to re-think and test with that setup